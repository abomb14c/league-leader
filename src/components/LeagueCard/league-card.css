.league-card-container {
  background-color: #3B3738;
  box-shadow: 0px 2px 10px 0px rgba(0, 0, 0, 0.30);
  display: flex;
  height: 300px;
  margin-left: 5%;
  margin-top: 3%;
  width: 25%;
}

.league-card-header {
  border-bottom: 2px solid #C63D0F;
  display: flex;
  color: #FDF3E7;
  font-weight: 200;
  height: 50px;
  padding-bottom: 5px;
  width: 100%;
}

.league-card-header-des{
  font-weight: 200;
  margin-left: 2%;
  margin-right: 2%;
}

.league-card-header-content {
  margin-right: 2%;
  color: #C63D0F;
}

.league-card-header-des-title {
  margin-left: 2%;
  margin-right: 3%;
  color: #C63D0F;
}