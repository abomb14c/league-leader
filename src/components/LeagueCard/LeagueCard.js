import React from 'react'; 
import './league-card.css';

const LeagueCard = ({admin, bet, league_type, name }) => {
  return (
    <div className="league-card-container">
      <div className="league-card-header">
        <h3 className="league-card-header-des-title">{name}</h3>
        <h3 className="league-card-header-des">Sport:</h3>
        <h3 className="league-card-header-content">{league_type}</h3>
        <h3 className="league-card-header-des">Wager:</h3>
        <h3 className="league-card-header-content">{bet}</h3>
      </div>
    </div>
  );
};

export default LeagueCard;