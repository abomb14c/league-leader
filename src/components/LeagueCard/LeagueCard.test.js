import React from 'react';
import { shallow } from 'enzyme';
import LeagueCard from './LeagueCard';

describe('LeagueCard', () => {
  let wrapper;
  let mockProps; 

  beforeEach(() => {
    mockProps = {
      admin: 2,
      bet: 'jersey',
      league_type: 'EPL',
      name: 'League'
    };

    wrapper = shallow(<LeagueCard
      {...mockProps}
    />);
  });
  
  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});