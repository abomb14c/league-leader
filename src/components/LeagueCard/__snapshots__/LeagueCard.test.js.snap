// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LeagueCard should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <LeagueCard
    admin={2}
    bet="jersey"
    league_type="EPL"
    name="League"
/>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="league-card-header"
>
        <h3
                className="league-card-header-des-title"
        >
                League
        </h3>
        <h3
                className="league-card-header-des"
        >
                Sport:
        </h3>
        <h3
                className="league-card-header-content"
        >
                EPL
        </h3>
        <h3
                className="league-card-header-des"
        >
                Wager:
        </h3>
        <h3
                className="league-card-header-content"
        >
                jersey
        </h3>
</div>,
      "className": "league-card-container",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h3
            className="league-card-header-des-title"
>
            League
</h3>,
          <h3
            className="league-card-header-des"
>
            Sport:
</h3>,
          <h3
            className="league-card-header-content"
>
            EPL
</h3>,
          <h3
            className="league-card-header-des"
>
            Wager:
</h3>,
          <h3
            className="league-card-header-content"
>
            jersey
</h3>,
        ],
        "className": "league-card-header",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "League",
            "className": "league-card-header-des-title",
          },
          "ref": null,
          "rendered": "League",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Sport:",
            "className": "league-card-header-des",
          },
          "ref": null,
          "rendered": "Sport:",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "EPL",
            "className": "league-card-header-content",
          },
          "ref": null,
          "rendered": "EPL",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Wager:",
            "className": "league-card-header-des",
          },
          "ref": null,
          "rendered": "Wager:",
          "type": "h3",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "jersey",
            "className": "league-card-header-content",
          },
          "ref": null,
          "rendered": "jersey",
          "type": "h3",
        },
      ],
      "type": "div",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="league-card-header"
>
          <h3
                    className="league-card-header-des-title"
          >
                    League
          </h3>
          <h3
                    className="league-card-header-des"
          >
                    Sport:
          </h3>
          <h3
                    className="league-card-header-content"
          >
                    EPL
          </h3>
          <h3
                    className="league-card-header-des"
          >
                    Wager:
          </h3>
          <h3
                    className="league-card-header-content"
          >
                    jersey
          </h3>
</div>,
        "className": "league-card-container",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <h3
              className="league-card-header-des-title"
>
              League
</h3>,
            <h3
              className="league-card-header-des"
>
              Sport:
</h3>,
            <h3
              className="league-card-header-content"
>
              EPL
</h3>,
            <h3
              className="league-card-header-des"
>
              Wager:
</h3>,
            <h3
              className="league-card-header-content"
>
              jersey
</h3>,
          ],
          "className": "league-card-header",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "League",
              "className": "league-card-header-des-title",
            },
            "ref": null,
            "rendered": "League",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Sport:",
              "className": "league-card-header-des",
            },
            "ref": null,
            "rendered": "Sport:",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "EPL",
              "className": "league-card-header-content",
            },
            "ref": null,
            "rendered": "EPL",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "Wager:",
              "className": "league-card-header-des",
            },
            "ref": null,
            "rendered": "Wager:",
            "type": "h3",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": "jersey",
              "className": "league-card-header-content",
            },
            "ref": null,
            "rendered": "jersey",
            "type": "h3",
          },
        ],
        "type": "div",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
